Library/ViewModels/CreditCardCellViewModel.swift

```diff
@@ -37,8 +37,14 @@ CreditCardCellViewModelOutputs, CreditCardCellViewModelType {
       .map(cardImage(with:))
 
     self.cardNumberAccessibilityLabel = self.cardProperty.signal.skipNil()
-      .map { [$0.type.description, Strings.Card_ending_in_last_four(last_four: $0.lastFour)]
-        .compactMap{ $0 }.joined(separator: ", ")
+      .map { card -> String in
+        guard let cardType = card.type else {
```

Comment:
The `guard` here should not be necessary...the `compactMap` makes sure to only concatenate (join) strings that are non optional.

So, unless I'm overlooking something this should be fine..

```
return [$0.type?.description, Strings.Card_ending_in_last_four(last_four: $0.lastFour)]
          .compactMap { $0 }
          .joined(separator: ", ")
```

Reply:
Yes you're right!